### DISCUSSION TITLE
Can recipes be moved between spaces?

### DISCUSSION THREAD

**Discussion by tehniemer** [TandoorRecipes/recipes#2598] [General]
I have some recipes that I would like to move from one space to another, I don't see an obvious way to do it, can this be done or do they have to be re-imported in the other space? 

If recipes can be moved, can they be moved in bulk or does it have to be one at a time?

**Comment by smilerz** [comment]
It cannot be moved - closest you can get is exporting recipes from a space and importing them into the new space.

**Reply by tehniemer** [reply]
Darn, is there a way to bulk select recipes for export? Or does that have to be done individually?

**Reply by smilerz** [reply]
`/export/` allows you to export multiple selected recipes or all recipes or recipes included in saved search.

on a side note: @vabene1111 - I don't see any way to navigate to export.

**Reply by tehniemer** [reply]
I found it in the little breifcase thing in the upper right corner. 
![image](https://github.com/TandoorRecipes/recipes/assets/11021632/5937b197-1680-4930-bd0c-24fd66dd4f1c)

**Reply by smilerz** [reply]
thanks - I should have known that

**Reply by tehniemer** [reply]
This is a bit harder that I would have hoped, I don't know the names of the recipes I want to export without seeing them, and it looks like the menu only displays 10 at a time. Instead of a dropdown menu is there a way to see a list of all the recipes on one page with checkboxes for selection? This would be a nice way to do bulk operations to existing recipes as well, like tagging or deleting.

**Reply by smilerz** [reply]
There are not any multiselect options at this time. 
One method to get at the result you are looking for would be:

- Create a Cookbook. 
- Add all of the recipes you want to export to that Cookbook. 
- Search for recipes in that book and save that search (this creates a 'Custom Filter'). 
- Go to the export page and select that CustomFilter to export.

**Reply by tehniemer** [reply]
I was able to create the filter as you suggested, but it doesn't seem to actually filter anything in the export menu.

**Reply by smilerz** [reply]
It doesn't filter the recipe dropdown - it allows you to export everything in the filter.
Note: there is an [open issue](#2158) related to exports

**Reply by tehniemer** [reply]
It doesn't seem to be working correctly even so. If I set the filter and hit download I get an empty zip file, if I check the "all recipes" button with the filter it downloads everything, not just the filtered recipes.

**Reply by smilerz** [reply]
can you open an issue that using the custom filter to export isnt' working correctly.

**Reply by tehniemer** [reply]
Done, #2601
